mirascope/core/base/from_call_args.py

```diff
@@ -0,0 +1,23 @@
+class FromCallArgs:
+    """
+    If you want to pass the function arguments to the response model, you can use this metadata.
+
+    example:
+    ```python
+    class Emotions(BaseModel):
+    sentences: Annotated[list[str], FromCallArgs()]
+    emotions: list[str]
+
+    @model_validator(mode="after")
+    def validate_output_length(self) -> Self:
+        if len(self.sentences) != len(self.emotions):
+            raise ValueError("length mismatch...")
+        return self
+
+    @openai.call(... same as before ..., response_model=Emotions)
```

Comment:
this should be a real example rather than the mock from the issue. also use shorthand instead of the prompt template